src/test/scala/com/google/cloud/spark/bigquery/DirectBigQueryRelationSuite.scala

```diff
@@ -79,35 +79,39 @@ class DirectBigQueryRelationSuite
 
   test("valid filters") {
     val validFilters = Seq(
-      GreaterThan("foo", "aardvark"),
       EqualTo("foo", "manatee"),
+      GreaterThan("foo", "aardvark"),
+      GreaterThanOrEqual("bar", 2),
       LessThan("foo", "zebra"),
       LessThanOrEqual("bar", 1),
```

Comment:
I know I wrote this, but it seems weird to test type mismatches like this are valid even if they are

Reply:
Does Spark know at compile time what the types of the various columns are? Would such a filter ever be provided to the DirectBigQueryRelation, in other words?